### ISSUE TITLE
appimage ?

### ISSUE THREAD

**Issue by datamaskinen** [keshavbhatt/olivia#89]
anyone having an appimage on this?

**Comment by keshavbhatt** [comment]
nah, am not able to pack mpv binary in the appimage of olivia, hence it won't work. mpv is the backend audio player olivia uses.
